app/models/queue_item.rb

```diff
@@ -0,0 +1,40 @@
+class QueueItem < ActiveRecord::Base
+  before_create :set_position
+  after_destroy :reassign_positions
+
+  belongs_to :user
+  belongs_to :video
+  validates_uniqueness_of :video_id, scope: :user_id
+  validates_uniqueness_of :position, scope: :user_id
+  validates_presence_of :user
+
+  delegate :title, to: :video, prefix: :video
+
+  def rating
+    review = Review.where(creator: user, video: video).first
+    review.rating if review
+  end
```

Comment:
What i checked in rails console is if the Review.where(...) in line 14 doesn't find any matching review, the "review" variable will be set to nil instead of false or any other (truthy) value. So the method will return nil (line#15 never runs returning the result of line#14). That seems to be working in the my_queue page, i mean it shows a blank selection for the ratings dropdown for that queue_item.